003-examples.md

```diff
@@ -1,36 +1,35 @@
-Examples: Mapping VO-DML ⇒ VOTable
-==================================
+# Examples: Mapping VO-DML to VOTable #
 
 This section shows examples of mappings as they may occur in realistic
 VOTables, using example Data Models. We will describe how the different
 VO-DML elements must be serialized and how annotations employing the
-&lt;VODML&gt; element can help one interpret the VOTable. In the later
+<VODML> element can help one interpret the VOTable. In the later
 normative section we turn this around and explicitly list all legal
 annotations, their constraints and interpretation.
 
 In this section we list some mappings from VO-DML to VOTable. We use
 examples extracted from a sample model with sample instances described
 in the next section. These should be seen as an introduction to the
-complete and formal specification in section 6.7.3 on how one MUST use
+complete and formal specification in section \ref{6.7.3} on how one MUST use
 VODML annotation in VOTable to indicate mapping to VO-DML.
 
 In the next few subsections we provide some examples how this mapping
-can be performed. It starts with the **ObjectType** and then discusses
-its components, **Attribute**, **Reference** and **Collection**. The
+can be performed. It starts with the **`ObjectType`** and then discusses

```

Comment:
If we wanted **all** the instance of `ObjectType`, with this capitalization, I could write a simple script that does it. The problem is that for `Attibute` and `Reference` I am not sure we want to do that. We could define some kind of special markup, e.g. [VO ObjectType] but I can hardly see the advantage with respect to **`ObjectType`**, except that we might decide to change the formatting, but then again that could be done with some easy *Search and Replace*.